fenix/app/src/main/java/org/mozilla/fenix/shopping/state/ReviewQualityCheckState.kt

```diff
@@ -75,7 +75,21 @@ sealed interface ReviewQualityCheckState : State {
                 val productUrl: String,
                 val highlights: Map<HighlightType, List<String>>?,
                 val recommendedProductState: RecommendedProductState = RecommendedProductState.Initial,
-            ) : ProductReviewState
+            ) : ProductReviewState {
+
+                /**
+                 * Highlights to display in compact mode that contains first 2 highlights of the
+                 * first highlight type.
+                 */
+                val highlightsForCompactMode: Map<HighlightType, List<String>>? =
+                    highlights?.entries?.firstOrNull()?.let { entry ->
+                        mapOf(entry.key to entry.value.take(NUMBER_OF_HIGHLIGHTS_FOR_COMPACT_MODE))
```

Comment:
out of interest, for the highlights:

1. do we have a preference on how these are ordered, e.g. alphabetically
2. if so, are we expecting the api will have already sorted this?

Reply:
Good questions! I will have to get back to you on those. Once we know the order, we should update the fake data to follow the same order though. 📓 

Since this is the UIState, it doesn't matter here but it would, for the mapper that will map the object exposed via AC to this State.